diagrams.net/assets/stencils/office/servers.xml

```diff
@@ -1678,7 +1678,7 @@
         <fillstroke/>
     </foreground>
 </shape>
-<shape aspect="variable" h="59.17" name="Excahnge Client Access Server Role" strokewidth="inherit" w="51.75">
```

Comment:
Same here: https://github.com/jgraph/drawio/blob/bbe89f62e093c56735281e63ac42b3fccb07425d/src/main/webapp/stencils/office/servers.xml#L1681

Reply:
> Same here: https://github.com/jgraph/drawio/blob/bbe89f62e093c56735281e63ac42b3fccb07425d/src/main/webapp/stencils/office/servers.xml#L1681

My [pull request](https://github.com/jgraph/drawio/pull/3423) already mentioned above addresses this issue in the upstream repository, too.